### DISCUSSION TITLE
Do not backup media taken before given date

### DISCUSSION THREAD

**Discussion by Batwam** [immich-app/immich#13425] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hello, in my current setup, I have backing up pictures from my mobile, which are then saved in the immich library. This library is separate from my Pictures folder so I regularly backup the immich library onto my Pictures folder. However, having two copies of the pictures is quickly taking a lot of room so, I am now using my Pictures folder as an "External Library" so I can delete older pictures from the immich library.

However, when deleting pictures from the immich library, they get re-uploaded by the mobile app. In order to work around that, I have created a folder with older pictures which I am excluding. This works but it's a bit convoluted to select several hundreds of picture in the iOS Photos app.

I was wondering if, in addition to excluding based on a folder, it would be possible to add an option exclude any picture take before a given date (e.g. 31-dec-2024)?

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#4169
